article/Experiment/dimensionality.tex

```diff
@@ -0,0 +1,148 @@
+\subsection{Node degree experiment}\label{subsec:degree-experiment}
+In this experiment we investigated how the nodes within a certain range of node degree perform. 
+The node degree is the number of connections the node has in the original interaction graph.
+Our initial hypothesis was that the fewer connections a node had, the more it would benefit from a higher number of convolutions.
+However the results obviously confirm that this is not the case.
+The NDCG results can be seen for Amazon-Cell-Sport on \autoref{tab:Amazon-Cell-Sport-ndcg-evaluation} and \autoref{fig:ndcg-individual-evaluation-amazon-book}, Yelp2020 on \autoref{tab:yelp2020-ndcg-evaluation} and \autoref{fig:yelp2020-ndcg-individual-evaluation} and Amazon-Book on \autoref{tab:amazon-book-NDCG-evaluation} and \autoref{fig:amazon-cell-sport-individual-evaluation-ndcg}.
+The results have been evaluated within a certain node degree, and "All nodes" is where all nodes has been evaluated.
+The \textbf{bold} results are the best performing and the \underline{underlined} results are the second best performing.
+The rightmost column is the result of the best performing original LightGCN method, that was run with 1 to 5 convolutions and these results for NDCG and Recall can be found in Appendix \ref{app:average-lightgcn-results}.
+The tables and figures with the recall results can be found in Appendix \ref{app:individual-recall-degree-experiment}.
+The experiment for Amazon-Cell-Sport can be seen on \autoref{tab:Amazon-Cell-Sport-ndcg-evaluation} showcase that for nodes with less than 46 connections that $E^{(5)}$ is usually the best performing and nodes with more than 46 connections $E^{(5)}$ performs worse than other embeddings with a lower number of convolutions.
+It does however often perform second best for many node degree ranges.
+It is also worth noting that $E^{(3)}$, $E^{(4)}$ and $E^{(5)}$ consistently outperforms LightGCN with 5 convolutions.
+This could be because that utilizing $E^{(0)}$ and $E^{(1)}$ is harmful for performance, because 91 \% of all items only have between 1-5 connections and therefore the embeddings needs more convolution to be able to gain enough information in the rest of the graph.
```

Comment:
```suggestion
This could be because that utilizing $E^{(0)}$ and $E^{(1)}$ is harmful for performance, because 91 \% of all items only have between 1-5 connections and therefore the embeddings need more convolution to capture the collaborative signals.
```